processor/stream/approved-es-documents/testIntakeIntegrationErrors.approved.json

```diff
@@ -355,7 +355,7 @@
                     "type": "DbError"
                 },
                 "grouping_key": "c3868d6704b923014eaffea034e70a3d",
-                "id": "cdefab0123456789"
+                "id": "cdefab0123456780"
```

Comment:
forgot to mention: not related to this PR, but i'd also rename these folders from `approved-es-documents` to `test_approved-es_documents`.
`test` prefix so to be more explicit (it feels a bit weird to have code and data test folders mixed without a strong convention) and underscores just because we use underscores everywhere.

im not strong about this, just an observation. could be changed in another PR if appropiate